models/DataObject/ClassDefinition/Listing/Dao.php

```diff
@@ -31,19 +34,63 @@ public function load(): array
     {
         $classes = [];
 
-        $classesRaw = $this->db->fetchFirstColumn('SELECT id FROM classes' . $this->getCondition() . $this->getOrder() . $this->getOffsetLimit(), $this->model->getConditionVariables(), $this->model->getConditionVariableTypes());
+        $classesData = $this->db->fetchAllAssociative(
+            'SELECT * FROM classes' .
```

Comment:
a micro-optimization, to replace the `*` with `id,name`, the other column `definitionModificationDate` is not required

Reply:
Thanks, created a follow up https://github.com/pimcore/pimcore/pull/18617 , it includes this change and an improvement of $force parameter usage.